core/arch/aarchxx/mangle.c

```diff
@@ -223,7 +223,7 @@ insert_clear_eflags(dcontext_t *dcontext, clean_call_info_t *cci,
 /* Creates a memory reference for registers saved/restored to memory. */
 static opnd_t
 create_base_disp_for_save_restore(uint base_reg, uint first_reg, uint reg,
```

Comment:
If I'm not mistaken, the argument "reg" is no longer used, so it would be clearer to remove it.